Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -2058,3 +2059,9 @@ bool CLuaEngineDefs::EngineRestreamWorld(lua_State* const luaVM)
     g_pClientGame->RestreamWorld();
     return true;
 }
+
+bool CLuaEngineDefs::EngineReinitMarkers(lua_State* const luaVM)
```

Comment:
No need to return a bool here, if it's always `true`. Also if the lua_State* arg isn't used, it doesn't need to be specified. 

The argparse can handle `void [func]()`.